docs/pipelines.md

```diff
@@ -319,21 +285,21 @@ For example see this `Pipeline` spec:
         resource: my-repo
 ```
 
-In this `Pipeline`, we want to test the code before we build from it, but there
-is no output from `test-app`, so `build-app` uses `runAfter` to indicate that
-`test-app` should run before it, regardless of the order they appear in the
-spec.
+### Using the `retries` parameter
 
-#### retries
+For each `Task` in the `Pipeline`, you can specify the number of times Tekton
+should retry its execution when it fails. When a `Task` fails, the corresponding
+`TaskRun` sets its `Succeeded` `Condition` to `False`. The `retries` parameter
+instructs Tekton to retry executing the `Task` when this happens.
 
-Sometimes you need a policy for retrying tasks which have problems such as
-network errors, missing dependencies or upload problems. Any of those issues must
-be reflected as False (corev1.ConditionFalse) within the TaskRun Status
-Succeeded Condition. For that reason there is an optional attribute called
-`retries` which declares how many times that task should be retried in case of
-failure.
+If you know that there will be issues with network connectivitity or missing
```

Comment:
Nit: suggest `If you know, for example, that there will be`

I think it's worth making clear that these aren't the only two conditions under which a user might want to retry a Task.  Just the two examples we have on hand :)

Reply:
Good call - made it clear it's an example.